contrib/init/bitcoind.service

```diff
@@ -20,8 +20,7 @@ ExecStart=/usr/bin/bitcoind -daemon \
                             -datadir=/var/lib/bitcoind
 
 # Make sure the config directory is readable by the service user
-PermissionsStartOnly=true
-ExecStartPre=/bin/chgrp bitcoin /etc/bitcoin
+ExecStartPre=+/bin/chgrp bitcoin /etc/bitcoin
```

Comment:
Is this syntax available on all of the supported platforms?

Reply:
Is there list of supported platforms?

Irregardless if there is a list or not, I assume the answer to be yes. Older Linux distribution versions with older systemd, tend to only package older bitcoin versions. The syntax was deprecated with systemd 241, which is currently in 
Debian stable. Likely the replacement syntax was established multiple releases before that. And even considering the rare (and very unlikely) case that you run this new bitcoin version under a very old systemd version, then the distribution package maintainer can remedy this (or the user itself).